data_managers/data_manager_build_kraken_database/data_manager/kraken_database_builder.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<tool id="kraken_database_builder" version="1.0.0" tool_type="manage_data" name="Kraken">
+    <description>database builder</description>
+    <requirements>
+        <requirement type="package" version="0.10.5">kraken</requirement>
+        <requirement type="package" version="1.1.11">jellyfish</requirement>
+        <requirement type="set_environment">KRAKEN_SCRIPT_PATH</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Error" />
+    </stdio>
+    <version_command>kraken -version | awk '{print $NF}'</version_command>
+    <command>
+    <![CDATA[
+        #set $library_list = str($libraries).split(',')
+        mkdir kraken-database && cd kraken-database &&
+        kraken-build --threads \${GALAXY_SLOTS:-1} --download-taxonomy --db "${database_name}" &&
+        #for $library in $library_list:
+            kraken-build --threads \${GALAXY_SLOTS:-1} --download-library $library --db "${database_name}"  &&
+        #end for
+        #for $fasta in $fasta_add:
+            kraken-build --threads \${GALAXY_SLOTS:-1} --db "${database_name}" --add-to-library "$fasta.input_fasta"  &&
+        #end for
+        kraken-build --threads \${GALAXY_SLOTS:-1} --db "${database_name}" --build --kmer-len ${kmer_len} --minimizer-len ${minimizer_len}  &&
+        cd .. && python \$KRAKEN_SCRIPT_PATH/data_manager/make_json.py --db "${database_name}" --out "${out_file}"
+    ]]>
+    </command>
+    <inputs>
+        <param name="database_name" type="text" label="Name for this database" />
+        <param name="libraries" type="select" multiple="True" label="Select partial library to download">
+            <option value="bacteria">Bacteria</option>
+            <option value="plasmids">Plasmids</option>
+            <option value="viruses">Viruses</option>
+            <option value="human">Human</option>
+        </param>
+        <repeat name="fasta_add" label="FASTA File" title="Add FASTA files to the library">
+            <param name="input_fasta" type="data" format="fasta" label="Select history item" />
```

Comment:
I think you can remove the repeat tag here and simple use `multiple="True"` in the data parameter.